server/dao/apis.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dao
```

Comment:
Can this be part of the storage package? "DAO" is obscure and generic, and with "storage", "dao", and "filter" packages, it feels like we're slicing our storage code very thinly (I admit "storage" is also generic, but it's less obscure :) )

Reply:
SGTM - thanks for clarifying. With this as the end goal and since it's an internal restructuring, I think it's fine to proceed with this on main.